libraries/chain/genesis_intrinsics.cpp

```diff
@@ -55,6 +55,7 @@ const std::vector<const char*> genesis_intrinsics = {
    "set_blockchain_parameters_packed",
    "is_privileged",
    "set_privileged",
+   "set_transaction_resource_payer",
```

Comment:
This is not a genesis intrinsic. Model off of something like `set_proposed_producers_ex`

Reply:
Removed from here.